oblib/taxonomy_misc.py

```diff
@@ -277,29 +277,44 @@ def is_ref_part(self, ref_part):
             return False
 
 
-class TaxonomyDocstrings(object):
+class TaxonomyDocumentation(object):
     """
     Loads the documentation strings for each concept from solar_2018-03-31_r01_lab.xml
     """
     def __init__(self):
-        self._docstrings = self._load_docstrings()
+        self._documentation = self._load_documentation()
 
-    def _load_docstrings(self):
+    def _load_documentation(self):
         label_file = "solar_2018-03-31_r01_lab.xml"
         filename = os.path.join(constants.SOLAR_TAXONOMY_DIR, "core", label_file)
 
-        taxonomy = _TaxonomyDocstringHandler()
+        taxonomy = _TaxonomyDocumentationHandler()
         parser = xml.sax.make_parser()
         parser.setContentHandler(taxonomy)
         parser.parse(open(filename))
-        return taxonomy.docstrings()
+        return taxonomy._documentation
 
-    def docstrings(self):
+    def get_all_concepts_documentation(self):
         """
         Used to lookup all docstrings.
 
         Returns:
             A map of all docstrings with a value of an array of two elements; array element 0 is
             a xlink:label and array element 1 is a xlink:role.
         """
-        return self._docstrings
+        return self._documentation
+
+    def get_concept_documentation(self, concept):
+        """
+        Used to load
+
+        Args:
+            concept (str): A concept name to lookup.
+
+        Returns:
+            The documentation for the concept or None if not found.
+        """
+        if concept in self._documentation:
+            return self._documentation[concept]
+        else:
+            return None
```

Comment:
this is a nit that might trip up some environments, but there should be a newline at the end of the file.